src/main/java/billennium/tests/service/quiz/QuizServiceImpl.java

```diff
@@ -84,7 +85,7 @@ public void saveResult(ResultModel resultModel, QuizModel quizModel, String user
         executingQuiz.setResult(resultMapper.mapToResultFromResultModel(resultModel));
         executingQuiz.setQuizStatus(QuizStatus.DONE);
         executingQuiz.setResultDetails(detailsMapper.mapToResultFromResultModel(resultModel, executingQuiz));
-        executingQuizRepository.save(executingQuiz);
+        executingQuizRepository.save(executingQuiz);  // TODO czy ten save jest potrzebny ?
```

Comment:
ja jestem zwolennikiem by podawać :) to jest nieszkodliwa dodatkowa linijka kodu. Według mojej wiedzy nasz ukochany Hibernate ma w budowany mechanizm dirty checking,użycie metody save() nie daje żadnego efektu. Ale dla mnie to nieszkodliwa dodatkowa linijka kodu :)

Reply:
ja jak widzę tego typu duplikowanie funkcjonalności w jednym miejscu to wydaje mi się, że coś zostało użyte nieświadomie.  Pisząc kod nie powinno się dodawać czegoś na wszelki wypadek. chcesz zostawić repo.save() to wywal @Transactional lub na odwrót - bo to właśnie to powiela funkcjonalność zapisu.